spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -824,6 +824,38 @@ def name_id_version(format_urn)
           expect(response.status).to eq(200)
           expect(authn_context_class_ref).to eq(Saml::Idp::Constants::AAL2_AUTHN_CONTEXT_CLASSREF)
         end
+
+        it 'returns AAL2-HSPD12 authn_context when AAL2-HSPD12 is requested' do
```

Comment:
**question:** even though we're ultimately removing support for the `aal/3` AuthnContexts, is it worth adding similar tests here for that?

Reply:
yep, added AAL3 specs in a585ea98d